guides/web-apps/acceptance-testing-with-selenium/README.md

```diff
@@ -230,33 +116,50 @@ There is one common pitfall when writing Selenium tests. The test is executed wi
 It is a good practice to always wait:
```

Comment:
I still dislike the formatting in the below section. What we have there is 3 rules:

## Wait for a text element to be present before you check the content of that element
## Wait for a button to be present before you click on that button
## Wait for presence of an input field before typing in it and wait for text to be present in input

Each of them should have a proper heading, code snippet and explanatory text. Right now some of them are part of a bullet point list, some of them are part of an ordered list.